README.md

```diff
@@ -13,42 +11,67 @@ The following classes are needed to configure, in order to execute the module.
 ### DynamicAssurance
 #### Configuration
 The following authproc filter configuration options are supported:
-  * `entitlementWhitelist`: Required, an array of strings containing values that can get higher assurance.
-  * `attribute`: Optional, a string that defines the name of the attribute that will store the value of LoA. Defaults to `eduPersonAssurance`.
-  * `candidates`: Optional, an array of strings that contains the allowed LoA values.
-  * `defaultAssurance`: Optional, a string to use as the default value of LoA. Defaults to `https://www.example.org/low`.
-  * `defaultElevatedAssurance`: Optional, a string to use as the default value of LoA. Defaults to `https://www.example.org/Substantial`.
-  * `idpPolicies`: Optional, an array of stings that contains the allowed IdP Policies.
-  * `idpTags`: Optional, an array of stings that contains the allowed IdP Tags.
+  * `attribute`: _Optional_, a string that defines the name of the attribute that will store the value of LoA. Defaults to `eduPersonAssurance`.
+  * `attributeMap`: _Optional_,  a map whose `keys` identify attribute names whose values can be mapped to assurance values. Represented LoA attributes, of the authenticating user, contain any of the list of the allowed values, then the module will append these values into the State. Defaults to
+  ```php
+      $assuranceMap = array(
+        'eduPersonAssurance' => array(
+            '1.2.840.113612.5.2.2.1' => array(                  // Classic
+                'https://refeds.org/assurance/IAP/low',
+                'https://refeds.org/assurance/IAP/medium',
+            ),
+            '1.2.840.113612.5.2.2.5' => array(                  // MICS
+                'https://refeds.org/assurance/IAP/low',
+                'https://refeds.org/assurance/IAP/medium',
+            ),
+        ),
+        '/^https:\/\/refeds\.org\/assurance/m',                 // REFEDS passthrough values
+        '/^https:\/\/aarc-community\.org\/assurance/m',         // AARC passthrough values
+    );
+  ```
+  :warning: The configured Assurance Map array is merged recursively with the default one from above.
+  * `defaultAssurance`: _Optional_, a string to use as the default value of LoA. Defaults to `null`.
```

Comment:
Turn this into an array to support assurance frameworks such as RAF which specify multiple assurance component values. New definition:
```
`defaultAssurance`: _Optional_, an array containing assurance value(s) to set by default when no assurance information is available. Defaults to `array()`, i.e. no assurance values are added by default.
```

Reply:
Done